src/Login.css

```diff
@@ -0,0 +1,103 @@
+body, html {
+    margin: 0;
+    padding: 0;
+    overflow: hidden; 
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+      'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+      sans-serif;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+  }
+  
+  .login-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100vw; 
+    height: 100vh; 
+    background-image: url('./Assets/Band2.png');
+    background-size: cover;
+    background-position: center;
+  }
+
+  @media (max-width: 767px) {
+    .login-container {
+      background-image: url('./Assets/Band2.png');
+    }
+  }
+  @media (max-width: 767px) {
+    body, html {
+      overflow: auto;
+    }
+}
```

Comment:
Media queries should be at the bottom to make sure they're working correctly and follow standard conventions. The order of the media query matters.

Reply:
Refactored!